### DISCUSSION TITLE
External USB works fine to Catalina or Big Sur, Internal SSD not seen

### DISCUSSION THREAD

**Discussion by GastonBA1** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#21] [Q&A]
I follow everything here for a Dell Latitude E7470.
The issue I am having is that the internal SSD is not seen at all in the Disk utility, so I am not able to do all the steps to only boot in MacOS.
I used the SSD in a USB adapter, and is able to install all of it. But when I am trying to make it permanent, does not see the SSD internal, and the USB external drives are seen as internal.
This may be related to the fact that this guide was made for external or dual boot.
I want to generate the OSX on the .M2 SDD on a USB dongle to be transferred to inside the machine and made single boot. 
SOmeone could help?
I am following https://github.com/adityabakare/macOS-Dell-Latitude-E7470

**Comment by adityabakare** [comment]
No, this guide isn't for an external disk or anything. Many users including me are using it on their internal SSD.

As mentioned in the README.md, you need to format the internal SSD beforehand in order to display the partitions correctly in Disk Utility, because NTFS partitions are read-only in here. Maybe use something like GParted.

Use the EFI in a USB drive( typically 4GB+), and then follow the installation instructions.

Also moving this to discussions, since this isn't a problem with the EFI.

**Comment by fantuan2019** [comment]
主板BIOS里把硬盘模式设置为AHCI就可以解决此问题，默认为raid模式。
